src/main/java/OOP2/task0/Main.java

```diff
@@ -0,0 +1,30 @@
+package OOP2.task0;
+
+
+public class Main {
+    public static void main(String[] args) {
+        // добавим листы
```

Comment:
У java doc'ов есть определенный формат, он в первом конспекте в конце указан, если я не ошибаюсь. Что-то типа такого должно быть. Такие комменты лучше не писать, название методов сами за себя говорить должны. Комментарии допустимы, но в редких случаях. 

```
/**
 * @author      Firstname Lastname <address @ example.com>
 * @version     1.6                 (current version number of program)
 * @since       1.2          (the version of the package this class was first added to)
 */
```